TUTORIAL_USE.md

```diff
@@ -0,0 +1,118 @@
+# Using the existing code generators
+
+Java::Geci provides several out-of-the-box code generators. 
+In this tutorial we'll use the generator `Accessor` as an example, 
+but the things explained in this tutorial apply to every other generator.
+
+You use Java::Geci by using the `Geci` class in tests.
```

Comment:
Since we have Geci API interface, Geci @interface and Geci class, perhaps the fully qualified class name would eliminate some confusion here.

Reply:
Added the fully qualified name of the class.